src/katello/agent/katelloplugin.py

```diff
@@ -117,10 +117,11 @@ def setup_plugin():
     cfg = plugin.cfg()
     rhsm_conf = Config(RHSM_CONFIG_PATH)
     certificate = ConsumerIdentity.read()
+    cfg.messaging.cacert = rhsm_conf['rhsm']['repo_ca_cert'] % rhsm_conf['rhsm']
```

Comment:
Does this handle the string substitution from rhsm.conf?: 
repo_ca_cert = %(ca_cert_dir)skatello-server-ca.pem